### PR TITLE
Set the finding date for Acunetix360 from FirstSeenDate

### PR DESCRIPTION
**Pull request by coheigea**
On an import from Acunetix360 the finding date is set to the import date rather than from FirstSeenDate in the report

### CODE REVIEW

**Review by StefanFl — APPROVED** [review]

File: dojo/tools/acunetix360/parser.py [hunk 1]
```diff
@@ -63,6 +63,10 @@ def get_findings(self, filename, test):
             finding.unsaved_req_resp = [{"req": request, "resp": response}]
             finding.unsaved_endpoints = [Endpoint.from_uri(url)]
 
+            if item["FirstSeenDate"] is not None:
```
Comment (damiencarol) [inline dojo/tools/acunetix360/parser.py:66]:
```suggestion
            if item.get("FirstSeenDate"):
```

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: dojo/tools/acunetix360/parser.py (hunk 1 above)
Comment (coheigea) [reply dojo/tools/acunetix360/parser.py]: Fixed @damiencarol

**Comment by coheigea** [comment]
Please review again @damiencarol

**Review by Maffooch — APPROVED** [review]

File: dojo/tools/acunetix360/parser.py [hunk 2]
```diff
@@ -63,6 +63,10 @@ def get_findings(self, filename, test):
             finding.unsaved_req_resp = [{"req": request, "resp": response}]
             finding.unsaved_endpoints = [Endpoint.from_uri(url)]
 
+            if item["FirstSeenDate"]:
```
Comment (damiencarol) [inline dojo/tools/acunetix360/parser.py:66]:
```suggestion
            if item.get("FirstSeenDate"):
```

**Comment by Maffooch** [comment]
@damiencarol are you good with this one?

**Comment by damiencarol** [comment]
@coheigea sorry for the delay, days off  :D . **all good for me, this is a nice patch**

**Pull request merged**